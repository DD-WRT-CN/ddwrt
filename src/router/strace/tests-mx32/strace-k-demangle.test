#!/bin/sh
#
# Check strace -k symbol names demangling.
#
# Copyright (c) 2017-2018 The strace developers.
# All rights reserved.
#
# SPDX-License-Identifier: GPL-2.0-or-later

test_prog=../stack-fcall-mangled

. "${srcdir=.}"/strace-k.test
