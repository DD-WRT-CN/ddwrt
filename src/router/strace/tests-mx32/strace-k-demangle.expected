^chdir .*(__kernel_vsyscall )?(__)?chdir ns::f3\(int\) ns::f2\(int\) ns::f1\(int\) ns::f0\(int\) main
^SIGURG .*(__kernel_vsyscall )?(__)?kill ns::f3\(int\) ns::f2\(int\) ns::f1\(int\) ns::f0\(int\) main
